src/niftyone/pipelines/group.py

```diff
@@ -153,7 +156,7 @@ def _get_group_key(record: pd.Series) -> fo.Classification:
     return key
 
 
-@lru_cache()
+@lru_cache
```

Comment:
Nb this would break in 3.7. Since we no longer want to use 3.7, it's fine.

Reply:
I kind of made the decision to drop 3.7 support because of EOL, but we could continue to support it as long as fiftyone does. I think where it gets a little tricky (even with 3.8 since that is EOL next summer), is a lot of the dependent packages have already started to drop support.

Reply:
Dropping 3.7 makes sense to me 👍